src/ItemTemplates/PeekRelationshipTemplate/PeekRelationship.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.VisualStudio.Language.Intellisense;
+
+namespace $rootnamespace$
+{
+    /// <summary>
+    /// Represents a PeekRelationship matching $rootSafeItemName$. <see cref="IPeekRelationship"/>s
+    /// are unique identifiers that apply to a <see cref="IPeekableItemSourceProvider"/>.
+    /// </summary>
+    public class $rootSafeItemName$ : IPeekRelationship
+    {
+        private $rootSafeItemName$() { }
+
+        public const string RelationshipName = "$rootSafeItemName$";
```

Comment:
I see why you want this, but FooRelationship.RelationshipName is sort of mouthful. But alas, I can't come up with anything better.